modules/apps/frontend-theme/frontend-theme-dialect-style-guide-sample-web/src/main/resources/META-INF/resources/js/guides/TabsGuide.js

```diff
@@ -0,0 +1,231 @@
+			<ClayIcon symbol="hashtag" />
+		</span>
+
+		<div className="mr-2 text-paragraph">Tertiary</div>
+
+		<div>
+			<div>
+				<h5 className="d-inline">k</h5>
+
+				<h1 className="d-inline">999.9</h1>
+
+				<h5 className="d-inline">k</h5>
+			</div>
+
+			<div className="font-weight-bold text-paragraph-sm text-right">
+				12.5%
+			</div>
+		</div>
+	</div>
+);
+
+const TABS_ACTIVE_BAR_COMPONENTS = [
+	{
+		className: 'nav-underline-active-bar-top',
+		type: 'underline',
+	},
+	{
+		className: 'nav-secondary nav-secondary-active-bar-top',
+		type: 'secondary',
+	},
+	{
+		child: tertiaryTab,
+		className: 'nav-tertiary nav-tertiary-active-bar-top',
+		type: 'tertiary',
+	},
+];
+
+const TABS_DISABLED_COMPONENTS = [
+	{
+		className: 'nav-underline',
+		disabled: true,
+		type: 'underline',
+	},
+	{
+		className: 'nav-secondary',
+		disabled: true,
+		type: 'secondary',
+	},
+	{
+		child: tertiaryTab,
+		className: 'nav-tertiary',
+		disabled: true,
+		type: 'tertiary',
+	},
+	{
+		child: 'segment',
+		className: 'nav-segment ',
+		disabled: true,
+		type: 'segment',
+	},
+];
+
+const TABS_COMPONENTS = [
+	{className: 'nav-underline', type: 'primary'},
+	{className: 'nav-secondary', type: 'secondary'},
+	{
+		child: tertiaryTab,
+		className: 'nav-tertiary',
+		type: 'tertiary',
+	},
+	{className: 'nav-segment', type: 'segment'},
+];
+
+const TABS_ICONS_COMPONENTS = [
+	{className: '', icon: 'after', type: 'primary'},
+	{className: '', icon: 'before', type: 'primary'},
+	{className: 'nav-secondary', icon: 'after', type: 'secondary'},
+	{className: 'nav-secondary', icon: 'before', type: 'secondary'},
+	{
+		child: tertiaryTab,
+		className: 'nav-tertiary',
+		type: 'tertiary',
+	},
+	{
+		child: tertiaryTab,
+		className: 'nav-tertiary',
+		type: 'tertiary',
+	},
+	{className: 'nav-segment', icon: 'after', type: 'segment'},
+	{className: 'nav-segment', icon: 'before', type: 'segment'},
+];
+
+const TABS_VERTICAL_COMPONENTS = [
+	{className: 'nav-underline-vertical', type: 'primary'},
+	{className: 'nav-secondary nav-secondary-vertical', type: 'secondary'},
+	{
+		child: tertiaryTab,
+		className: 'nav-tertiary nav-tertiary-vertical',
+		type: 'tertiary',
+	},
+];
+
+const TABS_TYPES = [
+	{components: TABS_COMPONENTS, title: Liferay.Language.get('tabs')},
+	{
+		className: 'd-flex',
+		components: TABS_VERTICAL_COMPONENTS,
+		title: Liferay.Language.get('tabs-vertical'),
+	},
+	{
+		components: TABS_ICONS_COMPONENTS,
+		title: Liferay.Language.get('tabs-icons'),
+	},
+	{
+		components: TABS_ACTIVE_BAR_COMPONENTS,
+		title: Liferay.Language.get('tabs-active-bar-position'),
+	},
+	{
+		components: TABS_DISABLED_COMPONENTS,
+		title: Liferay.Language.get('tabs-disabled'),
+	},
+];
+
+const TabsGuide = () => {
+	const TABS = (tab) => {
+		const [activeTabKeyValue, setActiveTabKeyValue] = useState(0);
+		const TABS_NAME = ['Tab 1', 'Tab 2', 'Tab 3'];
+
+		return (
+			<>
+				<ClayTabs className={tab.className} modern>
```

Comment:
Except for primary tabs we need to have `modern={false}`. Passing the modern prop adds the `nav-underline` className.

Reply:
Fixed